build/release/root/etc/notices/COPYRIGHT

```diff
@@ -1,5 +1,6 @@
 Open Maryland, not Indiana Operating System (OmniOS)
 Copyright 2016 OmniTI Computer Consulting, Inc. All rights reserved.
+Copyright 2017 OmniOS Community Edition Association (OmniOSce)
```

Comment:
maybe it should be

Copyright 2017 OmniOS Community Edition (OmniOSce), Association. All rights reserved.